src/azure/azureController.ts

```diff
@@ -117,48 +117,64 @@ export class AzureController {
 		await this.prompter.promptSingle(tenantQuestion, true);
 	}
 
-	public async getTokens(profile: ConnectionProfile, accountStore: AccountStore, settings: AADResource): Promise<ConnectionProfile> {
+	public async getAccount(accountStore: AccountStore): Promise<IAccount> {
```

Comment:
1. The method is actually doing add account operation, right? if so we should rename the method.
2. it isweird that I have to pass in the `accountStore` object, add account to it and then return the account. the AccountStore should be globally accessible.

Reply:
this method was already in the code I just divided to two methods. The previous one would add account and generate token. Now there are methods one add Account and other generates token